data/json/items/gun/9mm.json

```
@@ -522,11 +523,11 @@
     "looks_like": "hk_mp5",
     "type": "GUN",
     "reload_noise_volume": 10,
-    "name": { "str_sp": "Kel-Tec SUB-2000" },
-    "description": "A uniquely designed pistol caliber carbine with an integral folding stock.  It makes use of 9x19mm Glock magazines.",
+    "name": { "str_sp": "Kel-Tec Sub-2000" },
```

```suggestion
    "name": { "str_sp": "Kel-Tec SUB-2000" },
```
Kel-Tec uses allcaps https://www.keltecweapons.com/firearm/rifles/sub2000/